### PR TITLE
Feat/module from url

### PR DESCRIPTION
**Pull request by nicow-vi**
This PR allows to also pass the module as a web url

### CODE REVIEW

File: gatsby-node.js [hunk 1]
```diff
@@ -1,22 +1,28 @@
...
   if (skipOnPostBuild) {
     return;
   }
 
-  const stencil = require(module + "/hydrate");
-  const isIncrementalBuild = existsSync(CACHE_FILE_PATH)
-
   console.log(module);
 
+  const stencil = isUrl(module) ? await requireFromWeb(module + "/hydrate/index.js") : require(module + "/hydrate");
```
Comment (patrickhaug) [inline gatsby-node.js]: maybe we want to pass the complete url to the plugin? so we do not assume structure is _hydrate/index.js_

Comment (patrickhaug) [reply gatsby-node.js]: Plus it is more explicit in the gatsby app what we pass through.

File: gatsby-node.js [hunk 2]
```diff
@@ -19,7 +19,6 @@ exports.onPostBuild = async (_, pluginOptions) => {
   }
 
   console.log(module);
```
Comment (patrickhaug) [inline gatsby-node.js]: We can get rid of this as well :)

File: gatsby-node.js (hunk 1 above)
Comment (nicow-vi) [reply gatsby-node.js]: Was also thinking about that, but that would change the current implementation of the plugin. The original also assumes /hydrate. If we change that that means we have to also change all the gatsby repos where it is used to pass the module with the correct path.

File: gatsby-node.js [hunk 3]
```diff
@@ -1,22 +1,28 @@
...
   const { module, renderToStringOptions, skipOnPostBuild } = pluginOptions;
 
   if (skipOnPostBuild) {
     return;
   }
 
-  const stencil = require(module + "/hydrate");
-  const isIncrementalBuild = existsSync(CACHE_FILE_PATH)
-
   console.log(module);
```
Comment (patrickhaug) [inline gatsby-node.js]:
```suggestion
```

**Review by patrickhaug — APPROVED** [review]

**Review by ctmm — APPROVED** [review]

**Pull request merged**